generators/app/templates/static/build/pipelines-setup.md

```diff
@@ -47,9 +47,17 @@ The easiest way to add this pipeline to a project is to use the [callaway yeoman
 3. `cd` to project
 4. run `yo ccc`
 
-**NOTE:** When running from windows, the `build/*.sh` files do not seem to retain their executable permission. You'll need to figure out a way to [make the files linux executable](https://unix.stackexchange.com/questions/256964/transferring-executable-files-from-windows-to-linux-via-winscp) for the pipeline ci to run.
+**NOTE:** When running from Windows, the `build/*.sh` files do not seem to retain their executable permission. You'll need to figure out a way to [make the files linux executable](https://unix.stackexchange.com/questions/256964/transferring-executable-files-from-windows-to-linux-via-winscp) for the pipeline ci to run.
 
-Alternately, you could copy the [build folder](https://github.com/ChuckJonas/generator-ccc/tree/master/generators/app/templates/static/build) & [bitbucket-pipelines.yml](https://github.com/ChuckJonas/generator-ccc/blob/master/generators/app/templates/static/bitbucket-pipelines.yml) to your project.
+The recommended method is to manually grant the files in the [build folder](https://github.com/ChuckJonas/generator-ccc/tree/master/generators/app/templates/static/build) & [bitbucket-pipelines.yml](https://github.com/ChuckJonas/generator-ccc/blob/master/generators/app/templates/static/bitbucket-pipelines.yml) permission to execute.
+
+**Only needed if running from windows** Run the following git commands:
+
+1. `git update-index --chmod=+x build/schedule.sh`
```

Comment:
can this command just be run recursively on the folder?  

`git update-index --chmod=+x build`

Otherwise we gotta remember to update these instructions if we add a new build file

Reply:
Afraid not:

> git update-index --chmod=+x build 
error: build: is a directory - add individual files instead
fatal: Unable to process path build

Reply:
I bet there is some way to do it though. Asking in ineedhelp.

Reply:
https://stackoverflow.com/questions/12288212/git-update-index-assume-unchanged-on-directory

Something like this might work (although probably doesn't work in windows)